src/spyglass/lfp/lfp_merge.py

```diff
@@ -125,3 +70,22 @@ def get_lfp_object(key: dict):
             return LFPV1 & lfp_object.fetch("KEY")
         else:
             return ImportedLFPV1 & (LFPOutput.ImportedLFP & key).fetch("KEY")
+
+    def fetch_nwb(self, *attrs, **kwargs):
```

Comment:
Would this benefit from a secondary key `"upstream"` or something along those lines? The thinking being that it would specify the part table (e.g. `"LFPV1"`) to be referenced in the parent table entry that could be fetched. Not as clean, but potentially more specific?

Reply:
Eric and I will be testing this more thoroughly today. My hope is that someone will be able to do the following without issue: `(MergeTable & "arbitrary_key='value'").fetch_nwb()`

It's not clear why `*attrs, **kwargs` are in the class method but not accepted by the actual util. Just future proofing?